src/app/zap-templates/zcl/data-model/chip/washer-controls-cluster.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="WasherControlFeature" type="BITMAP32">
+    <cluster code="0x0053"/>
+    <field mask="0x01" name="Spin"/>
+    <field mask="0x02" name="Rinse"/>
+  </bitmap>
+  
+  <struct name="SpinSpeedIndex">
+    <cluster code="0x0053"/>
+    <item fieldId="0" name="Label" type="char_string" length="64" isNullable="false" writable="false"/>
+    <item fieldId="1" name="SpinSpeed" type="int8u" isNullable="false" writable="false"/>
+  </struct>
```

Comment:
Reading ahead to the PR that removes SpinSpeedIndex, you don't need this anymore.
```suggestion
```